optuna/samplers/_cmaes.py

```diff
@@ -60,6 +60,9 @@ def objective(trial):
             study.optimize(objective, n_trials=20)
 
     Please note that this sampler does not support CategoricalDistribution.
+    However, DiscreteUniformDistribution (`suggest_discrete_uniform`) and
+    Int(Log)Distribution (`suggest_int`) are supported.
```

Comment:
To generate links to related optuna pages, how about this?

```suggestion
    However, :class:`~optuna.distributions.DiscreteUniformDistribution`
    (:func:`~optuna.trial.Trial.suggest_discrete_uniform`) and
    Int(Log)Distribution (:func:`~optuna.trial.Trial.suggest_int`) are supported.
```

Reply:
Good idea, pushed.

Reply:
Thanks!